### PR TITLE
Bug 1919387: Wait till the DOM is loaded when writing to the body localStorage issue

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @spadgett

### CODE REVIEW

File: frontend/public/tokener.html [hunk 1]
```diff
@@ -2,33 +2,35 @@
...
+          ['exp', 'email', 'name', 'userID'].forEach(function (key) {
+            localStorage.setItem(key, json[key]);
+          });
+        } catch (e) {
+          error = e;
+          console.error(e);
+          document.body.append('localStorage must be enabled:');
+          document.body.append(e.message || e.toString());
+        }
+        if (!error) {
```
Comment (spadgett) [inline frontend/public/tokener.html:19]: Digging into the code, I don't believe the `localStorage` items are strictly required to continue, particularly for OpenShift where we user information is not stored there. We track login state in a cookie, and user preferences have moved to a config map. We might just continue and not fail on an error.

Comment (jhadvig) [reply frontend/public/tokener.html:19]: @spadgett so do we want to keep the `onload()` method? Or just check if the `document.body` is not `null` when trying to write into it ?

**Comment by spadgett** [comment]
/override ci/prow/ceph-storage-plugin

The ceph-storage-plugin job should only have run on PRs with changes to `frontend/packages/ceph-storage-plugin`. It failed due to an unrelated CI config change. Overriding status on this PR.

**Review by spadgett — APPROVED** [review]
/lgtm

Let's go ahead with this fix for now, and we can consider trying to continue even after the error in a follow up.

**Pull request merged**